src/parcsr_ls/par_amg_setup.c

```diff
@@ -3034,10 +3033,21 @@ hypre_BoomerAMGSetup( void               *amg_vdata,
          }
          max_eig_est[j] = max_eig;
          min_eig_est[j] = min_eig;
-         hypre_ParCSRRelax_Cheby_Setup(A_array[j],max_eig, min_eig,
-                                       cheby_fraction, cheby_order, scale, variant, &coefs, &ds);
+
+         cheby_ds[j]                             = hypre_SeqVectorCreate(hypre_ParCSRMatrixNumRows(A_array[j]));
+         hypre_VectorVectorStride(cheby_ds[j])   = hypre_ParCSRMatrixNumRows(A_array[j]);
```

Comment:
Is `hypre_VectorIndexStride` set? This should not be important here, but for completeness it is better to have.

Reply:
Done (https://github.com/hypre-space/hypre/pull/388/commits/ffaf12cba10e92fadc608004d0a59c58eac16453#diff-a3bfdae21bbae32a82a1636e33577f7e7041c83e4969183592df510c5d8898c0R3051)